pipelines/pingcap/tiproxy/latest/merged_sqllogic_test.groovy

```diff
@@ -0,0 +1,167 @@
+                                prow.checkoutRefs(REFS)
+                            }
+                        }
+                    }
+                }
+                dir("tidb-test") {
+                    cache(path: "./", filter: '**/*', key: "git/pingcap/tidb-test/rev-${REFS.pulls[0].sha}", restoreKeys: ['git/pingcap/tidb-test/rev-']) {
+                        retry(2) {
+                            script {
+                                component.checkoutV2('git@github.com:pingcap/tidb-test.git', 'tidb-test', "master", REFS.pulls[0].title, GIT_CREDENTIALS_ID)
+                            }
+                        }
+                    }
+                }
+            }
+        }
+        stage('Prepare') {
+            steps {
+                dir('tiproxy') {
+                    sh label: 'tiproxy', script: '[ -f bin/tiproxy ] || make'
+                }
+                dir('tidb-test') {
+                    cache(path: "./", filter: '**/*', key: "ws/${BUILD_TAG}") {
+                        sh "touch ws-${BUILD_TAG}"
+                        sh label: 'prepare thirdparty binary', script: """
+                        chmod +x download_binary.sh
+                        ./download_binary.sh --tidb=master --pd=master --tikv=master
+                        cp ../tiproxy/bin/tiproxy ./bin/
+                        ls -alh bin/
+                        ./bin/tidb-server -V
+                        ./bin/pd-server -V
+                        ./bin/tikv-server -V
+                        ./bin/tiproxy --version
+                        """
+                    }
+                }
+            }
+        }
+        stage('TestsGroup1') {
+            matrix {
+                axes {
+                    axis {
+                        name 'TEST_PATH_STRING'
+                        values 'index/between/1 index/between/10 index/between/100', 'index/between/100 index/between/1000 index/commute/10',
+                            'index/commute/100 index/commute/1000_n1 index/commute/1000_n2',
+                            'index/delete/1 index/delete/10 index/delete/100 index/delete/1000 index/delete/10000',
+                            'random/aggregates_n1 random/aggregates_n2 random/expr' , 'random/expr random/select_n1 random/select_n2',
+                            'select random/groupby'
+                    }
+                }
+                agent{
+                    kubernetes {
+                        namespace K8S_NAMESPACE
+                        yamlFile POD_TEMPLATE_FILE
+                        defaultContainer 'golang'
+                    }
+                }
+                stages {
+                    stage("Test") {
+                        options { timeout(time: 40, unit: 'MINUTES') }
+                        steps {
+                            dir('tidb-test') {
+                                cache(path: "./sqllogic_test", filter: '**/*', key: "ws/${BUILD_TAG}/tidb-test") {
```

Comment:
the cache key `ws/${BUILD_TAG}/tidb-test` different from before.
`ws/${BUILD_TAG}/tidb-test` --> `ws/${BUILD_TAG}`